pkgs/by-name/pg/pg_search/package.nix

```diff
@@ -0,0 +1,64 @@
+{
+  cargo-pgrx_0_15_0,
+  fetchFromGitHub,
+  lib,
+  postgresql,
+  rustfmt,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pg_search";
+  version = "0.19.5";
+
+  src = fetchFromGitHub {
+    owner = "paradedb";
+    repo = "paradedb";
+    rev = "v${version}";
+    hash = "sha256-/7AHKUUJW1wh39pUXlVEIykobE2IRMul3pWxoUCdKjA=";
+  };
+
+  cargoHash = "sha256-Yn4D8oaENExqzx1p/9OTEKsVnlJDitn1JST6rhHGPrw=";
+
+  preBuild = ''
+    export PGRX_BUILD_FLAGS="--frozen -j $NIX_BUILD_CORES"
+    export PGRX_HOME=$TMPDIR/.pgrx
+    cargo pgrx init --pg${lib.versions.major postgresql.version} ${postgresql.pg_config}/bin/pg_config
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+
+    cargo pgrx package \
+      --pg-config ${postgresql.pg_config}/bin/pg_config \
+      --package pg_search \
+      --out-dir $out
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mv $out/${postgresql}/* $out
+    rm -rf $out/nix
+
+    runHook postInstall
+  '';
+
+  nativeBuildInputs = [
+    cargo-pgrx_0_15_0
+    rustfmt
+  ];
+
+  PGRX_PG_SYS_SKIP_BINDING_REWRITE = "1";
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "The transactional Elasticsearch alternative built on Postgres";
+    homepage = "https://www.paradedb.com";
```

Comment:
It would be good to add `meta.changelog` as well, to have that linked for automated updates.